CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 Changelog
 =========
 
+## TBD 
+
+* Removed the bugsnag android gradle plugin due to incompatibility with UE 5.1+[#220](https://github.com/bugsnag/bugsnag-unreal/pull/220)
```

Comment:
We should add information / a link about how to do this manually

Reply:
Done 👍